### PR TITLE
Update to 9p2 SDK

### PR DESCRIPTION
**Pull request by vzarytovskii**

### CODE REVIEW

**Review by T-Gro — APPROVED** [review]

**Comment by vzarytovskii** [comment]
This should be good to go after we merge VS fixes, since it can _technically_ fail signed CI builds.

**Review by psfinaki — APPROVED** [review]

File: global.json
Comment (psfinaki) [inline global.json:1]: Leaving this to prevent accidental merge

**Pull request merged**
